tesseract_kinematics/include/tesseract_kinematics/ikfast/impl/ikfast_inv_kin.hpp

```diff
@@ -201,6 +224,7 @@ const std::string& IKFastInvKin::getBaseLinkName() const { return base_link_name
 const std::string& IKFastInvKin::getTipLinkName() const { return tip_link_name_; }
 const std::string& IKFastInvKin::getName() const { return name_; }
 const std::string& IKFastInvKin::getSolverName() const { return solver_name_; }
+std::vector<std::vector<double>> IKFastInvKin::getFreeJointCombos() const { return free_joint_combos_; };
```

Comment:
If you change `generateAllPossibleCombinations` to be a static function, I don't think there would be a need for this getter function anymore since all combinations would have already been passed to the constructor

Reply:
Removed again.